datafusion/sqllogictest/test_files/ddl.slt

```diff
@@ -607,6 +607,19 @@ CREATE TABLE table_without_values(field1 BIGINT, field2 BIGINT);
 statement error Execution error: 'IF NOT EXISTS' cannot coexist with 'REPLACE'
 CREATE OR REPLACE TABLE IF NOT EXISTS table_without_values(field1 BIGINT, field2 BIGINT);
 
+# CREATE OR REPLACE
```

Comment:
Can you also please test:
1. Using `CREATE OR REPLACE` when the table doesn't already exist
2. Using `CREATE OR REPLACE` to replace the same name with a different definition (e.g. a parquet file rather than CSV) and show it is different?